aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -144,4 +165,12 @@ and will generate HTML as follows.
 
 (of course the domain, in the case above is localhost, but this will be substituted for whatever the actual domain hosting the web site is hosted at).
 
+- - -
+
+## Additional Resources
+
+* [Areas](../../../../controllers/areas.md)
```

Comment:
This is one of the reasons I prefer the `xref` link format. However, I have some questions to ask. I'll open a new issue for that ... this isn't a good spot for *CrazyRex:tm:* quesitons.